### PR TITLE
Relayer: Send meta info of L1 interactions in many data items

### PR DESCRIPTION
**Pull request by janekolszak**

### CODE REVIEW

File: src/relay/tags.go [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+import (
+	"encoding/base64"
+	"strconv"
+
+	"github.com/jackc/pgtype"
+	"github.com/warp-contracts/syncer/src/utils/bundlr"
+	"github.com/warp-contracts/syncer/src/utils/model"
+)
+
+func getTags(payload *Payload, source, env string, interaction *model.Interaction, random []byte) (out []bundlr.Tag) {
```
Comment (szynwelski) [inline src/relay/tags.go:12]: The `Contract-Tx-Id` and `Tx-Id` tags should also be set in this function. For both L1 and L2.

File: src/relay/arweave_meta_bundler.go [hunk 2]
```diff
@@ -137,38 +86,30 @@ func (self *ArweaveMetaBundler) createMetaDataItem(payload *Payload, arweaveBloc
 
 func (self *ArweaveMetaBundler) fill(payload *Payload) (err error) {
 	for blockIdx, arweaveBlock := range payload.ArweaveBlocks {
-		var (
-			dataItem      *bundlr.BundleItem
-			dataItemBytes []byte
-		)
-
 		items := make([]*bundlr.BundleItem, len(arweaveBlock.Interactions))
```
Comment (szynwelski) [inline src/relay/arweave_meta_bundler.go:89]: Is this array needed?

File: src/relay/arweave_meta_bundler.go [hunk 3]
```diff
@@ -69,62 +67,13 @@ func (self *ArweaveMetaBundler) createDataItem(arweaveBlock *ArweaveBlock, idx i
...
-		{Name: "Env", Value: self.Config.Relayer.Environment},
-		// Block specific
-		{Name: "Arweave-Block-Height", Value: strconv.FormatUint(arweaveBlock.Message.BlockInfo.Height, 10)},
-		{Name: "Arweave-Block-Timestamp", Value: strconv.FormatUint(arweaveBlock.Message.BlockInfo.Timestamp, 10)},
-		{Name: "Arweave-Block-Hash", Value: arweaveBlock.Message.BlockInfo.Hash},
-		{Name: "Sequencer-Height", Value: strconv.FormatInt(payload.SequencerBlockHeight, 10)},
-		{Name: "Sequencer-Timestamp", Value: strconv.FormatInt(payload.SequencerBlockTimestamp, 10)},
-	}
+	// Arweave specific
+	out.Tags = append(out.Tags,
```
Comment (szynwelski) [inline src/relay/arweave_meta_bundler.go:73]: Move to `getTags` function

Comment (janekolszak) [reply src/relay/arweave_meta_bundler.go:73]: Done

File: src/relay/tags.go (hunk 1 above)
Comment (janekolszak) [reply src/relay/tags.go:12]: Done

**Review by szynwelski — APPROVED** [review]

**Pull request merged**